### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by m13hall**

### CODE REVIEW

File: db/migrate/20160504191325_make_phone_numbers_big_enough_for_phone_numbers.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class MakePhoneNumbersBigEnoughForPhoneNumbers < ActiveRecord::Migration
+  def change
+  	change_column :users, :phonenumber, :integer, limit: 8
```
Comment (schmitzc) [inline db/migrate/20160504191325_make_phone_numbers_big_enough_for_phone_numbers.rb]: ❓ Would it make more sense to store the phone number as a string? You aren't using the phone number for any calculations.

Comment (soojison) [inline db/migrate/20160504191325_make_phone_numbers_big_enough_for_phone_numbers.rb]: that would make things easier, but we've used the `number_to_phone` method to format the phone number nicely...

Comment (schmitzc) [inline db/migrate/20160504191325_make_phone_numbers_big_enough_for_phone_numbers.rb]: The `number_to_phone` method will work with a string, too. 😄  http://api.rubyonrails.org/classes/ActionView/Helpers/NumberHelper.html#method-i-number_to_phone

Comment (soojison) [inline db/migrate/20160504191325_make_phone_numbers_big_enough_for_phone_numbers.rb]: @m13hall then things just got a lot easier!

**Pull request merged**